website/website-gen/src/main.rs

```diff
@@ -39,8 +40,38 @@ fn build_docs() -> Result<()> {
     Ok(())
 }
 
+fn build_wasm_demo() -> Result<()> {
+    // install wasm-pack if not available
+    let res = Command::new("wasm-pack").arg("--version").status();
+    if res.is_err() {
+        let status = cargo().args(&["install", "wasm-pack"]).status()?;
```

Comment:
I would suggest downloading `https://github.com/rustwasm/wasm-pack/releases/download/v0.8.1/wasm-pack-v0.8.1-x86_64-unknown-linux-musl.tar.gz` instead and saving a lot of time when it is not cached. You can even easily check the version.

Reply:
The flip side is that we'd have to implement our own caching logic, to avoid download on every local build. I think it makes sense to defer these concerns to `cargo install` and travis own caching.